src/components/ScrollShadows.tsx

```diff
@@ -0,0 +1,103 @@
+import { useLayoutEffect } from "@react-aria/utils";
+import { ReactNode, useCallback, useRef, useState } from "react";
+import { Css, Palette, Properties } from "src";
+
+interface ScrollShadowsProps {
+  children: ReactNode;
+  /** Allows for styling the container */
+  xss?: Properties;
+  /** Set to true if the container scrolls horizontally */
+  horizontal?: boolean;
+  /** Defines the background color for the shadows */
+  bgColor?: Palette;
+}
+export function ScrollShadows(props: ScrollShadowsProps) {
+  const { children, xss, horizontal = false, bgColor = Palette.White } = props;
+  const { height = "auto", width = "auto" } = xss ?? {};
+  // This is admittedly extremely hacky. It expects the background color to be in the format "rgba(255, 255, 255, 1)".
+  // If we ever change how we define our color palette in Beam, then this will break and will need to be fixed.
```

Comment:
Suggestion: Makes sense, maybe we can detect a non-rgba value and throw an error? Np if that's not worth it.

Reply:
Yea, I like that. I will update.